src/Provider/GoogleMapsPlaces/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Change Log
+
+The change log describes what is "Added", "Removed", "Changed" or "Fixed" between each release.
+
+## 4.0.0
```

Comment:
I know our management of starting release number is confusing but you should start with version `0.1.0`.

Reply:
Ahh, ok. I noticed a lot of providers start from 4.x for some reason. Fixed.